src/test_l1_plat_power.c

```diff
@@ -229,19 +231,19 @@ void test_l1_plat_power_negative_PLAT_TERM (void)
 {
     gTestID = 4;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
-	pmStatus_t ret;
+    pmStatus_t ret;
 
     // Variation/Step 01: Call PLAT_TERM() - close interface without initializing
     ret = PLAT_TERM();
     CHECK_FOR_EXTENDED_ERROR_CODE( ret, PWRMGR_NOT_INITIALIZED, PWRMGR_SUCCESS);
```

Comment:
change all tabs spaces to spaces... address in whole file

Reply:
Done